.github/workflows/codeql.yml

```diff
@@ -26,9 +26,6 @@ jobs:
       contents: read
       security-events: write
 
-    strategy:
-      fail-fast: false
-
```

Comment:
Opportunistic, after Andony noticed this is meaningless without a language matrix.